scripts/copy.js

```diff
@@ -0,0 +1,75 @@
+const fs = require("fs");
+const util = require("util");
+
+const readDir = util.promisify(fs.readdir);
+const readfile = util.promisify(fs.readFile);
+
+const rawImageBaseUrl =
+  "https://raw.githubusercontent.com/Real-Dev-Squad/website-static/main/members";
+
+async function fileWalker(dir) {
+  const readMemberDir = readDir(dir);
+  const personFolderNames = await readMemberDir.then(
+    (personFolder) => personFolder
+  );
+  const userDataPromises = personFolderNames.map((person) =>
```

Comment:
Why mixing the nomenclature? `Person` or `User`, let's try and choose one please